sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/rel/BeamEnumerableConverter.java

```diff
@@ -207,27 +217,7 @@ private static void runCollector(PipelineOptions options, BeamRelNode node) {
     runCollector(options, node);
 
     Collector.globalValues.remove(id);
-    return values.stream().collect(Collectors.toList());
-  }
-
-  private static Enumerable<Object> collect(PipelineOptions options, BeamRelNode node) {
-    long id = options.getOptionsId();
-    Queue<Row> values = new ConcurrentLinkedQueue<>();
-
-    checkArgument(
-        options
-            .getRunner()
-            .getCanonicalName()
-            .equals("org.apache.beam.runners.direct.DirectRunner"),
-        "SELECT without INSERT is only supported in DirectRunner in SQL Shell.");
```

Comment:
Checking for the direct runner is gone. Did this move somewhere else?

Reply:
it's in `collectRows()`